cmd/api/config.go

```diff
@@ -55,12 +55,17 @@ type config struct {
 			KeepFiles int  `default:"5"` // number of files to keep
 		}
 	}
-	TelemetryPublisher struct {
-		Enabled            bool   `default:"false"`
-		MetricsHubURL      string `default:""`
-		MetricsHubAPIKey   string `default:""`
-		PublishingInterval string `default:"10s"`
-	}
+	TelemetryPublisher TelemetryPublisherConfig
+}
+
+// TelemetryPublisherConfig contains configuration attributes for the telemetry module.
+type TelemetryPublisherConfig struct {
+	Enabled            bool   `default:"false"`
+	MetricsHubURL      string `default:""`
+	MetricsHubAPIKey   string `default:""`
+	PublishingInterval string `default:"10s"`
+
+	ChainStackCollectFrequency string `default:"15m"`
```

Comment:
This `ChainStackCollectFrequency` is a new configuration. It specifies how often the last executed block number per chain is collected. The name is more generic since the same telemetry sub-module can be used to also collect more info about each chain-stack.